libs/auth-api-lib/src/lib/types.ts

```diff
@@ -1,5 +1,7 @@
 export const DEFAULT_DOMAIN = '@island.is'
 
+export const defaultAcrValue = 'eidas-loa-high'
```

Comment:
I think we should also create a AcrEnum with the eidas value and islandis passkey and this default value should be 
```
export const defaultAcrValue = AcrValue.EidasLoaHigh
```
 or something like that.

Reply:
Sounds good, fixed.